framework/src/modules/fee/constants.ts

```diff
@@ -12,9 +12,10 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-export const MODULE_ID_FEE = 14;
-export const NATIVE_TOKEN_CHAIN_ID = 0;
 export const defaultConfig = {
-	feeTokenID: '0000000000000000',
 	minFeePerByte: 1000,
 };
+
+export const CONTEXT_STORE_KEY_AVAILABLE_FEE = 'CONTEXT_STORE_MIN_FEE_KEY';
```

Comment:
This is a super tiny and almost irrelevant remark, that I almost feel bad about reporting it 😅

It looks like all other constants have their string value equal to the constant name. It would be good to keep it consistent also here. This probably got out of sync during automated constant rename.